docs/plugins/transform-class-constructor-call.md

```diff
@@ -6,6 +6,65 @@ permalink: /docs/plugins/transform-class-constructor-call/
 package: babel-plugin-transform-class-constructor-call
 ---
 
+Allows to call a class constructor
```

Comment:
Actually I think we want to put a deprecation message here since the proposal was dropped. https://github.com/babel/babel/wiki/Babel-7